src/pages/articles/bootstrap/bootstrap-buttons/index.md

```diff
@@ -2,14 +2,35 @@
 title: Bootstrap Buttons
 ---
 ## Bootstrap Buttons
+The Bootstrap framework provides you with various styling options for buttons. These styles help you provide a visual representation to the user of what the button may do.
 
-This is a stub. [Help our community expand it](https://github.com/freecodecamp/guides/tree/master/src/pages/articles/bootstrap/bootstrap-buttons/index.md).
+#### How To Use:
+To use bootstrap buttons you follow the same steps that you would to create a button in HTML except you also apply the applicable CSS class to the button. A code example has been provided below.
 
-[This quick style guide will help ensure your pull request gets accepted](https://github.com/freecodecamp/guides/blob/master/README.md).
+**Code Example:**
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+`<button type="button" class="btn btn-primary">Primary</button>`
```

Comment:
You will have access to bootstrap styles here. Including an actual HTML button with the primary class will help this article.

There is no special syntax for including HTML in markdown, you can write HTML and markdown in the same file :+1: